docs/learn/storage/aws-ami.md

```diff
@@ -0,0 +1,121 @@
+# Kaia Endpoint Node AMIs
+
+Kaia offers an AWS AMI ([Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html)) for Kaia endpoint nodes (EN). These AMIs are pre-installed with both the EN software and the chaindata, enabling users to quickly and conveniently launch a fully operational node. With Kaia's AMI, setting up a new endpoint node on demand becomes seamless, allowing for rapid scaling and simplifying the process of joining the Kaia network.
+
+## Types of EN AMIs
+
+Kaia provide different types of AMIs, whose chaindata has been synced with different modes.
+
+| **Type** | **Sync Mode** | **AMI Name** |
+|----------|---------------|--------------|
+| full             | Full mode                                         | `kaia-xxxx-clean-full-en-xxxx` |
+| pruning          | Full mode, live pruning enabled                   | `kaia-xxxx-clean-pruning-en-xxxx` |
+| state-migrated   | Full mode, state migrated (or state batch pruned) | `kaia-xxxx-clean-en-xxxx`|
+| archive          | Archive mode                                      | `kaia-xxxx-clean-archive-en-xxxx` |
+
+Kaia provide these 4 types of AMIs for both mainnet and kairos.
+
+See [State Batch Pruning](../state-pruning/#state-batch-pruning-state-migration) for more details on state migrated chaindata.
+See [Block Syncronization](./block-sync.md) for more details on block sync modes.
```

Comment:
```suggestion
See [Block Synchronization](../../learn/storage/block-sync.md) for more details on block sync modes.
```
The path change is supposing that 'aws-ami.md' file is moved under the Node Quick Reference section